Makefile

```diff
@@ -7,13 +7,23 @@ help:
 start: ## Spin up the containers and run the app UI in watch mode
 	docker compose up -d
 
-stop: ## Shut down the containers
+stop: ## Stop the containers
+	docker compose stop \
+	&& rm -f docker/redis/data/dump.rdb
+
+status: ## Status the containers
+	docker compose ps
+
+down: ## Shut down the containers
 	docker compose down \
 	&& rm -f docker/redis/data/dump.rdb
 
+down-with-vol: ## Shut down the containers and removed volumes
```

Comment:
No need for a new command, you can add ARGS to the other command like we do for other commands so the args can be used

Reply:
I added it because the command already defined in full makes it easier to use by the IDE, thus giving more proactivity and efficiency.

If you think you shouldn't have it, we can remove it.

Reply:
It really makes sense to pass on as an argument, considering that for productivity and efficiency the removal of the volume of data does not enter into this requirement, since such an action requires greater attention.

I'll leave it as an argument as suggested